homework-3/controllers/users.js

```diff
@@ -0,0 +1,156 @@
+const pool = require('../data-access/connection.js');
```

Comment:
you still have only one layer
Let's create two separate modules: `user.service` and `user.repository` for business logic and data access layer